scss/appNav.module.scss

```diff
@@ -7,3 +7,21 @@
   margin-left: 2em;
   margin-right: 2em;
 }
+
+.nav-buttons {
+  flex-shrink: 0;
+  & > * {
+    min-width: 5em;
+    @media (min-width: 992px) {
+      background-color: white !important;
+    }
+  }
+}
+
+.dropdown-item {
+  color: #838383;
```

Comment:
This color `#838383` is used really often. Should this be setup in our theme instead?

Reply:
I created new file colors.module.scss from which we can import default colors. It wasn't possible to import from index (it uses global styles). 

TO-DO. Change all colors in scss files to import from this new file.